### PR TITLE
Updated README.md

### LINKED ISSUE #2: 🚀 [Feature] : Better README.md
**Is your feature request related to a problem? Please describe.**
The current `README.md` lacks detailed information, making it difficult for new users to understand the project's purpose, how to set it up, and how to contribute. This can be frustrating and may hinder adoption and collaboration.

**Describe the solution you'd like**
I would like a comprehensive `README.md` that includes the following sections:
- Project Overview: A brief description of the project and its goals.
- Features: A list of key features and functionalities.
- Installation: Step-by-step instructions on how to set up the project locally.
- Usage: Examples and explanations of how to use the project.
- Contributing: Guidelines for contributing to the project.
- License: Information about the project's license.
- Contact: How to get in touch with the project maintainers.

**Describe alternatives you've considered**
An alternative could be to create a separate documentation website for detailed guides and tutorials, but enhancing the `README.md` is a more immediate and accessible solution.

**Additional context**
Including badges for build status, dependencies, and other relevant information could also enhance the `README.md`. Screenshots or additional visual aids may be helpful to illustrate key points.

_If you want to speed up the process of this issue, please send 🚀 as a reaction._

### LINKED ISSUE #9: 🚀 [Feature] : Update README.md due to langx/website
Please add README following sections and make it better readibility as well.

I took from langx/website repo https://github.com/langx/website/blob/main/README.md?plain=1

Can you please update this readme.md as well like the langx/website readme.md

```md

## Table of Contents
- [Getting Started](#getting-started)
- [Contributing](#contributing)
- Stats ...
  - Stargazers
  - Contributors
- [License](#license)

## Getting started

To clone this project, simply run the following command:


# clone this repo
git clone  https://github.com/langx/website

## Contributing

We welcome contributions from the community! If you'd like to contribute to LangX, please fork our repository and submit a pull request. We'll review your changes and merge them if they meet our guidelines. Thank you for helping to make LangX even better!

We'll review your changes and merge them if they meet our guidelines. Thank you for helping to make LangX even better!

LangX is an open source project and we welcome contributions from the community. If you're interested in contributing, please check out our GitHub repository for more information.

## Versioning

LangX uses [Semantic Versioning](https://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/langx/website/releases).

## Stats

### Stargazers

[![Stargazers over time](https://starchart.cc/langx/website.svg?variant=adaptive)](https://starchart.cc/langx/website)

### Contributors

[![GitHub Contributor Over Time](https://contributor-overtime-api.git-contributor.com/contributors-svg?chart=contributorOverTime&repo=langx/token-website)](https://git-contributor.com?chart=contributorOverTime&repo=langx/token-website)

[![GitHub Contributors Image](https://contrib.rocks/image?repo=langx/token-website)](https://github.com/langx/token-website/graphs/contributors)

## License

LangX is released under the [MIT License](./LICENSE). If you use this project, please include the license file in your distribution.

```

_If you want speed up process of this issue, please send 🚀 as a reaction._

### PR DESCRIPTION
**Pull request by Caluryth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,89 @@
...
+```
+
+3. Navigate to the project directory
+
+```bash
+# navigate to the projects directory
+cd token-website
+```
+
+4. Install dependencies
```
Comment (xuelink) [inline README.md]: No need to have this step, its only HTML and CSS

File: README.md [hunk 2]
```diff
@@ -1,11 +1,89 @@
...
+
+```bash
+npm install
+```
+
+## Start Development Server
```
Comment (xuelink) [inline README.md]: No need this section as well

File: README.md [hunk 3]
```diff
@@ -1,11 +1,89 @@
...
+npm run dev
+
+# or start the server and open the app in a new browser tab
+
+npm run dev -- --open
+```
+
+## Issues
+
+If you encounter any issues, ensure that your [Node.js](https://nodejs.org/en) and [npm](https://www.npmjs.com/) versions are up to date.
```
Comment (xuelink) [inline README.md]: Please mention here https://github.com/langx/token-website/issues to open new issues instead of Nodejs and NPM links

**Review by xuelink — CHANGES_REQUESTED** [review]
Awesome, you did a great job, please attach here new commits by fixing this review then i am going to merge this PR.

File: README.md (hunk 1 above)
Comment (Caluryth) [reply README.md]: So, remove the dependencies step? (Not the cd token-website part though, right?)

File: README.md (hunk 3 above)
Comment (Caluryth) [reply README.md]: Okay!

File: README.md (hunk 1 above)
Comment (xuelink) [reply README.md]: Only the 4.th part.

Comment (Caluryth) [reply README.md]: @xuelink  - When I make edits, should I close this PR, do the edits, and then open a new PR?

File: README.md [hunk 4]
```diff
@@ -1,11 +1,71 @@
...
+
+## Setup
+
+### Prerequisites
+
+Before you begin, please make sure you have met the following requirements.
+
+- You have installed [Node.js](https://nodejs.org/en) and [npm](https://www.npmjs.com/) (Node Package Manager).
+- You have a [GitHub](https://github.com/) account
+
```
Comment (xuelink) [inline README.md]: No need this section as well

File: README.md [hunk 5]
```diff
@@ -1,11 +1,71 @@
...
+Welcome to the LangX Token webpage! Our app, in addition to helping you learn a new language through practice and immersion with native speakers from around the world, will give you the opportunity to earn with our unique "Learn-to-Earn" initiative. This is where you can find out everything you need to know about this initiative. 
 
-Our "Coming Soon" page has been released! The full project is still under development and the first major release has not yet been launched.
+This webpage is built with HTML, CSS and JavaScript.
 
-Please stay tuned for updates and progress.
+## Table of Contents
 
-We appreciate your patience and interest.
+- [Prerequisites](#prerequisites)
```
Comment (xuelink) [inline README.md]: No need this line.

File: README.md [hunk 6]
```diff
@@ -1,11 +1,71 @@
...
+- [Getting Started](#getting-started)
+- [Development Server](#start-development-server)
```
Comment (xuelink) [inline README.md]: No need this line, please update table of contents as well

**Review by xuelink — CHANGES_REQUESTED** [review]
Small updates pls again.

**Review by xuelink — APPROVED** [review]
Great Job!

**Pull request merged**